deploy/crds/api.astarte-platform.org_astartes_crd.yaml

```diff
@@ -14820,6 +14820,21 @@ spec:
                     type: string
                   image:
                     type: string
+                  maxOfflineMessages:
+                    description: The maximum number of QoS 1 or 2 messages to hold
+                      in the offline queue. Defaults to 1000. Set to -1 for no maximum
```

Comment:
1000 is really small. Honestly I think we should go for 1000000.
@rbino does AppEngine receives any error when we reach the maximum?

Reply:
Same as above